src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -920,4 +921,36 @@ public void cannotCheckOutWhenItemIsNotLoanable()
       hasItemBarcodeParameter(nod),
       hasLoanPolicyParameters(notLoanablePolicy))));
   }
+
+  @Test
+  public void cannotCheckOutWhenItemLimitOfLoanPolicyIsReached()
```

Comment:
Please add test that outlines scenario:
1. There are two loan policies (e.g. for dvd and text material types)
2. Limit values are 1 and 10 respectively
3. A patron has borrowed 1 dvd and 1 text
4. Test patron cannot borrow dvd and can borrow text

Reply:
done